utils/mezoura/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=mezoura
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2022-02-01
+PKG_SOURCE_URL:=https://github.com/linosgian/mezoura
+PKG_SOURCE_VERSION:=ae8969d207f9f68535cc9f5cabdf8345226352e2
+PKG_MIRROR_HASH:=3cd82bf5cf5754aaffbb7656fc3e0c525ac925f42a70eb23aa3004d1e2542096
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-2.0
+PKG_MAINTAINER:=Linos Giannopoulos <linosgian00@gmail.com>
```

Comment:
This order is not good. Please take a look how it is done in other Makefiles.

1. Also, you can fix the deprecated SPDX License Identifier

Reply:
pushed a fixup for this. Let me know if I got it right